cypress/e2e/requests.cy.js

```diff
@@ -0,0 +1,22 @@
+describe("Requests Page", () => {
+  it("A logged out user should not able to see the requests list.", () => {
+		// Visit a protected route in order to allow cypress to set the cookie and mock the login
+		cy.visit("/requests")
+
+		cy.get('div.alert-heading').contains('Unauthorized').then(() => {
+			cy.log("A logged out user is not able to view requests.");
+		});
+	})
+  
+	it("A logged in user should be able to see the requests list.", () => {
+		// Call the custom cypress command to log in
+		cy.login(Cypress.env('TEST_SCIENTIST_USER'), Cypress.env('TEST_SCIENTIST_PW'))
+
+		// Visit a protected route in order to allow cypress to set the cookie and mock the login
+		cy.visit("/requests")
+
+		cy.get('h1').contains('My Requests').then(() => {
```

Comment:
also, this change in the component library needed to happen to avoid selecting things in tests with bootstrap classes:
https://github.com/scientist-softserv/webstore-component-library/pull/160